src/EnergyPlus/AirflowNetworkSolver.cc

```diff
@@ -3289,6 +3299,310 @@ Label90: ;
 	}
 
 	void
+	AFEOAF(
```

Comment:
@mjwitte These names are original code and used as arguments every function. I prefer to change names late as an issue.